pkg/compiler/compiler.go

```diff
@@ -188,16 +171,6 @@ func (c *ModelCompiler) loadModelMetaData(path string) error {
 	return nil
 }
 
-func (c *ModelCompiler) loadPluginVersion(path string) error {
```

Comment:
why was this removed ?

Reply:
because it was not being used in any template - Charles had removed the plugin `VERSION` file in the sd-fabric models, and this was throwing an error - I started investigating where is was used and it's not